paper.tex

```diff
@@ -473,7 +473,7 @@ \section*{Introduction}
 working in the open can be great,
 but can also unintentionally suppress voices.
 Programming Historian makes an ombudsperson available for private chats and facilitation
-to ensure that no one is excluded.
+to ensure that no one is excluded. Software Carpentry operates by a Code of Conduct that outlines acceptable standards of behavior for our community members and those interacting with the Carpentries at in-person events and in virtual spaces. Community members on a 'Policy Subcommittee' serve as advocates for the CoC and adjudicate reported Code of Conduct violations. Together, the Code of Conduct and the Policy Subcommittee help ensure that all members of the community are treated with respect.
```

Comment:
This looks good - you'll just need to remove the "CoC" abbreviation and write out Code of Conduct like you have elsewhere.